app/celery/tasks.py

```diff
@@ -391,6 +393,9 @@ def update_letter_notifications_to_error(self, notification_references):
     )
 
     current_app.logger.info("Updated {} letter notifications to technical-failure".format(updated_count))
+    notifications = dao_get_notifications_by_reference(references=notification_references)
+    for notification in notifications:
+        send_delivery_status_to_service.apply_async([notification.id], queue=QueueNames.NOTIFY)
```

Comment:
You will need to wrap notification with str(notification_id)

Reply:
changed in 3594fc.